coreos-koji-tagger/Dockerfile

```diff
@@ -10,6 +10,12 @@ RUN dnf update -y && dnf clean all
 # Install pagure/fedmsg libraries
 RUN dnf -y install dnf-plugins-core python3-libpagure fedora-messaging koji krb5-workstation && dnf clean all
 
+# Update koji src to 1.18 for multicall support (not yet packaged as an RPM)
+RUN dnf -y install git && dnf clean all
+RUN rm -rf /usr/lib/python3.7/site-packages/koji
+RUN git -C /opt/ clone https://pagure.io/koji.git && git -C /opt/koji checkout koji-1.18.0
```

Comment:
Minor: that's `-b koji-1.18.0` :)

Reply:
i'm checking out the `koji-1.18.0` tag. Not trying to create a new branch or anything:

```
Note: checking out 'koji-1.18.0'.                                                                                  
                                                                                                                   
You are in 'detached HEAD' state. You can look around, make experimental                                                                                                                                                              
changes and commit them, and you can discard any commits you make in this                                                                                                                                                             
state without impacting any branches by performing another checkout.                                                                                                                                                                  
                                                                                                                                                                                                                                      
If you want to create a new branch to retain commits you create, you may                                                                                                                                                              
do so (now or later) by using -b with the checkout command again. Example:                                                                                                                                                            
                                                                                                                   
  git checkout -b <new-branch-name>                                                                                
                                                                                                                                                                                                                                      
HEAD is now at c7657331 PR#1588: 1.18 release
```

Reply:
I meant `clone -b`, not `checkout -b`.

Reply:
ok I see. So if I `git clone -b koji-1.18.0` I won't need the `&& git checkout koji-1.18.0`